devops/jobs/CreateSandbox.groovy

```diff
@@ -202,6 +202,8 @@ class CreateSandbox {
 
                 stringParam("security_group","sandbox-vpc","")
 
+                stringParam("instance_initiated_shutdown_behavior","terminate","")
```

Comment:
Do we even want this to be a user parameter?  I can't think of a reason why @edx/devops can you come up with one?
I'd either just code it into ansible-provision or send it as an Environment from here unless someone thinks of a reason to let folks override the behavior.